samples/wifi/provisioning/common/proto-c/common.pb.c

```diff
@@ -0,0 +1,21 @@
+/* Automatically generated nanopb constant definitions */
+/* Generated by nanopb-0.4.6 */
+
+#include "common.pb.h"
+#if PB_PROTO_HEADER_VERSION != 40
+#error Regenerate this file with the current version of nanopb generator.
+#endif
+
+PB_BIND(ScanParams, ScanParams, AUTO)
+
+
+PB_BIND(WifiInfo, WifiInfo, AUTO)
+
+
+
+
+
+
+
+
+
```

Comment:
is nanopb really generating a file with so many empty lines ?

Reply:
Use nanopb in ncs to generate the code again, and the output are the same.